libraries/dotnet/EnergyOrigin.Setup/EnergyOrigin.Setup/EnergyOrigin.Setup.csproj

```diff
@@ -0,0 +1,37 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFramework>net8.0</TargetFramework>
+		<ImplicitUsings>disable</ImplicitUsings>
+		<Nullable>enable</Nullable>
+		<IsPackable>true</IsPackable>
+		<PackageId>EnergyOrigin.Setup</PackageId>
+		<RepositoryUrl>https://github.com/Energinet-DataHub/energy-origin</RepositoryUrl>
+		<Authors>Energinet</Authors>
+		<Company>Energinet DataHub A/S</Company>
+		<PackageDescription>This library contains functions for setting up Energy Origin solutions with logging, swagger, api versioning and OpenTelemetry</PackageDescription>
+		<PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<FrameworkReference Include="Microsoft.AspNetCore.App" />
+	</ItemGroup>
+
+	<ItemGroup>
+	  <PackageReference Include="Asp.Versioning.Mvc" Version="8.0.0" />
+	  <PackageReference Include="Asp.Versioning.Mvc.ApiExplorer" Version="8.0.0" />
+	  <PackageReference Include="MassTransit" Version="8.1.3" />

```

Comment:
I dont believe all our systems are using masstransit. Do we still want it here or could we make some kind of builder pattern on the metrics/traces so it is optional

Reply:
We keep it to have masstransit tracing on on default. They will just be empty if there's no masstransit in the project